axum/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "axum"
-version = "0.8.0" # remember to bump the version that axum-extra depends on
+version = "0.8.1" # remember to bump the version that axum-extra depends on
```

Comment:
Can you explain why we need this change? Is it to prepare the next release?

Reply:
Yeah, I want to publish right after this PR to update the crates.io page.